src/main/resources/db/migration/V1__Init_tables.sql

```diff
@@ -0,0 +1,141 @@
+DROP TABLE IF EXISTS cc_employee CASCADE;
+DROP TYPE IF EXISTS cc_employee_status;
+DROP TYPE  IF EXISTS cc_employee_role;
+
+CREATE TYPE cc_employee_status as enum ('INACTIVE','ACTIVE','ARCHIVED');
+CREATE TYPE cc_employee_role as enum ('STAFF','ADMIN');
+
+CREATE TABLE cc_employee
+(
+    id         BIGINT                   NOT NULL PRIMARY KEY,
+    email      VARCHAR(100)             NOT NULL,
+    password   CHAR(64)                 NOT NULL,
+    first_name VARCHAR(100)             NOT NULL,
+    last_name  VARCHAR(100)             NOT NULL,
+    role       cc_employee_role         NOT NULL,
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    status     cc_employee_status       NOT NULL
+);
+
+COMMENT ON COLUMN cc_employee.email IS 'Employee''s email as login username';
+COMMENT ON COLUMN cc_employee.password IS 'Login password(hashcode)';
+COMMENT ON COLUMN cc_employee.first_name IS 'Employee''s first name';
+COMMENT ON COLUMN cc_employee.last_name IS 'Employee''s last name';
+COMMENT ON COLUMN cc_employee.role IS 'Employee''s role: STAFF(default)/ADMIN';
+COMMENT ON COLUMN cc_employee.created_at IS 'Employee account create time, eg:2022-09-14T19:10:25.083Z';
+COMMENT ON COLUMN cc_employee.updated_at IS 'Employee account last update time, eg:2022-09-14T19:10:25.083Z';
+COMMENT ON COLUMN cc_employee.status IS 'Employee status: INACTIVE(default)/ACTIVE/ARCHIVED';
+
+DROP TABLE IF EXISTS franchisee CASCADE;
+DROP TYPE IF EXISTS franchisee_status;
+
+CREATE TYPE franchisee_status as enum ('UNVERIFIED','VERIFIED');
```

Comment:
Same as above, do we really need it at db level? Besides, if the statuses includes only these two, why don't we just use `is_verified` instead?